front/src/app/components/layouts/deal-list/view.tsx

```diff
@@ -102,26 +103,10 @@ export class DealListView extends React.PureComponent<IProps, any> {
             title: 'Stats',
             render: (price: string, record: IDeal) => {
                 return (
-                    <div className="sonm-deals-list__cell__stats">
-                        <div className="sonm-deals-list__cell__stats__name">
-                            CPU Count
-                        </div>
-                        <div className="sonm-deals-list__cell__stats__value">
-                            {record.benchmarkMap.cpuCount}
-                        </div>
-                        <div className="sonm-deals-list__cell__stats__name">
-                            GPU ETH hashrate
-                        </div>
-                        <div className="sonm-deals-list__cell__stats__value">
-                            {record.benchmarkMap.ethHashrate}
-                        </div>
-                        <div className="sonm-deals-list__cell__stats__name">
-                            RAM size
-                        </div>
-                        <div className="sonm-deals-list__cell__stats__value">
-                            {record.benchmarkMap.ramSize}
-                        </div>
-                    </div>
+                    <Benchmark
+                        data={record.benchmarkMap}
+                        keys={['cpuCount', 'ethHashrate', 'ramSize']}
```

Comment:
it is bad practice /
whats wrong here? i will not explain this error one more time.